messages/en.json

```
@@ -322,7 +323,7 @@
 	"generateReference": "Generate Transaction Reference",
 	"genericToastError": "Error",
 	"genericToastLoading": "Loading...",
-	"genericToastsuccess": "Success",
+	"genericToastSuccess": "Success",
```

Biome (2.1.2)

[error] The key genericToastSuccess was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)